File: app/src/main/java/com/cristhian/gametobeat/util/IgdbUtil.java
package com.cristhian.gametobeat.util;

public class IgdbUtil {

    public static final String BASE_URL = "https://api.igdb.com/v4/";
    private static final String BASE_IMAGE_URL = "https://images.igdb.com/igdb/image/upload/t_";

    private static final String IMAGE_URL_COVER_BIG = "cover_big";

    public static String getUrlImage(String imageId) {
        StringBuilder sb = new StringBuilder()
            .append(BASE_IMAGE_URL)
            .append(IMAGE_URL_COVER_BIG)
            .append("/")
            .append(imageId)
            .append(".jpg");

        return sb.toString();
    }
}

File: app/src/main/java/com/cristhian/gametobeat/util/IgdbUtil.kt
package com.cristhian.gametobeat.util

import com.cristhian.gametobeat.network.RetrofitClientInstance

object IgdbUtil {

    fun getUrlImage(imageId: String): String {
        val sb = StringBuilder()
                .append(RetrofitClientInstance.BASE_IMAGE_URL)
                .append(RetrofitClientInstance.IMAGE_URL_COVER_BIG)
                .append("/")
                .append(imageId)
                .append(".jpg")
        return sb.toString()
    }
}